app/controllers/application_controller.rb

```diff
@@ -455,6 +458,13 @@ def maintenance_warning
     @minutes_to_maintenance = NextMaintenance.minutes_to(now)
   end
 
+  def get_onboarding_status
+    if @is_current_community_admin
+      #TODO: maybe not with global variable?
+      @onboarding_status = Admin::OnboardingWizard.new(@current_community).setup_status
```

Comment:
The OnboardingWizard expects community_id, not a community object as ctor parameter. This works by accident now because ActiveRecord find_by is flexible and seems to know how to interpret an AR object as id in find calls. But that's an accidental implementation detail so this should be changed to pass in id instead of AR object.